persistqueue/sqlackqueue.py

```diff
@@ -113,16 +112,14 @@ def _mark_ack_status(self, key, status):
 
     @sqlbase.with_conditional_transaction
     def clear_acked_data(self):
-        sql = """DELETE FROM {table_name}
-            WHERE {key_column} IN (
-                SELECT _id FROM {table_name} WHERE status = ?
-                ORDER BY {key_column} DESC
-                LIMIT 1000 OFFSET {max_acked_length}
```

Comment:
Why not just delete this part? - Is it doing the same thing but slower as the new query?